.travis.yml

```diff
@@ -70,16 +70,8 @@ before_script:
   # Export web server URL for browser tests.
   - export SIMPLETEST_BASE_URL=http://localhost:8888
 
-  # Install PHPCS to check for Drupal coding standards.
-  - travis_retry composer global require drupal/coder:8.2.12
-  - ~/.composer/vendor/bin/phpcs --config-set installed_paths ~/.composer/vendor/drupal/coder/coder_sniffer
-
 script:
   # Run the PHPUnit tests which also include the kernel tests.
   - ./vendor/bin/phpunit -c ./core/phpunit.xml.dist ./modules/rules
-  # Check for coding standards - first show the sniffs that are used.
-  - cd modules/rules && ~/.composer/vendor/bin/phpcs -e --report-width=130
-  # Show the coding standards violations in detail.
-  - ~/.composer/vendor/bin/phpcs --colors -s --report-width=130
-  # Run again to give a summary and total count.
-  - ~/.composer/vendor/bin/phpcs --colors --report=summary --report-width=130
+  # Check for coding standards.
+  - cd modules/rules && ./vendor/bin/phpcs --colors --report-width=130
```

Comment:
Now that the coding is clean with no style warning I see that it is cleaner (and will run quicker) with just the single `phpcs` call. However, when I was working on the fixes it was invaluable to have the list of sniffs shown in the separate report, and for the summary report. It would be useful to leave these lines in but commented, so that when we next get discrepancies or the Coder sniffs change and we have failures, it will be easy for anyone to get the full info and work on fixes. I can make a patch for these little changes if you want me to.